### PR TITLE
Switch default docker-compose to PostgreSQL+Elasticsearch

### PR DESCRIPTION
**Pull request by sw-yx**
## What was changed

- added postgres + ES docker compose file
- promoted it to default, replacing the cassandra file

## Why?

we want people to have a better default experience. they dont really need the scalability of cassandra in docker compose, and we want to make it easy for them to check out custom searchable attributes

have discussed with both Samar and Ryland

## notes to reviewer

i'm not actually sure if i took care of all the details of adding ES. i just plug it in to `temporal-network` and temporal auto discovers it? is there anything else to take care of?

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -27,6 +44,7 @@ services:
       - temporal
     environment:
       - TEMPORAL_CLI_ADDRESS=temporal:7233
+      - TEMPORAL_PERMIT_WRITE_API=true
```
Comment (wxing1292) [inline docker-compose.yml:47]: what is this env?

**Review by wxing1292 — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (sw-yx) [reply docker-compose.yml:47]: this activates the Terminate button in temporal web, makes it full CRUD

Comment (sw-yx) [reply docker-compose.yml:47]: i think i pasted it in the wrong service

**Comment by sw-yx** [comment]
@alexshtin @wxing1292 this configuration doesnt seem very stable. even just leaving it open for a couple hours i have these scary segfaults

CleanShot 2021-08-11 at 18 22 59@2x

**Comment by sw-yx** [comment]
also when I try to use an elasticsearch feature on the new default docker-compose (the completely new one with postgres + ES) it gives me this failure which makes me think its not properly configured. hmm.

**Comment by alexshtin** [comment]
Crush apparently related to qemu on M1. And I fixed Elasticsearch configuration.

**Comment by clintonb** [comment]
> ...and we want to make it easy for them to check out custom searchable attributes

How? I'm finally checking out this feature, and it's unclear how to configure them via some form of Docker configuration.

**Pull request merged**